### PR TITLE
Issue 201

### PR DESCRIPTION
**Pull request by PatDyn**
Should Solve Issue #201 

* We now do some minifying and resource shrinking while trying to keep a readable stack trace. 
    * this is done by a minimal proguard-rules.pro config
* Update okhttp3 dep to work with current R8 versions 
* Added a string.toLongOrZero() extension function to avoid type errors
* implement a signing config, that checks wether some env variables are set
    * thus deciding which origin to use for passwords
* Added a basic debug build type

Note:
No automated tests were implemented for this issue.

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -76,15 +76,21 @@
     <string name="updating_database">Mise à jour des rapports… (<xliff:g id="currentSize">%1$d</xliff:g>/<xliff:g id="totalSize">%2$d</xliff:g>)</string>
 
     <!-- Trackers Fragment -->
-    <string name="trackers_status"><xliff:g id="percentage">%1$d</xliff:g>%% | <xliff:g id="apps">%2$d</xliff:g> apps</string>
+    <plurals name="trackers_status">
+        <item quantity="one"><xliff:g id="percentage">%1$d</xliff:g>%% | <xliff:g id="apps">%2$d</xliff:g> app</item>
+        <item quantity="many"><xliff:g id="percentage">%1$d</xliff:g>%% | <xliff:g id="apps">%2$d</xliff:g> apps</item>
```
Comment (Jean-BaptisteC) [inline app/src/main/res/values-fr/strings.xml:81]:
```suggestion
        <item quantity="other"><xliff:g id="percentage">%1$d</xliff:g>%% | <xliff:g id="apps">%2$d</xliff:g> apps</item>
```
Crash with many on release version

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -76,15 +76,21 @@
...
     <!-- TrackersDetail Fragment -->
     <string name="open_tracker_page">Ouvrir le site web du pisteur</string>
     <string name="detection_rules">Règles de détection</string>
     <string name="code_signature">Règle de détection (code) :</string>
     <string name="network_signature">Règle de détection (réseau) :</string>
     <string name="apps_list">Liste des applications</string>
-    <string name="trackers_presence">Présent dans <xliff:g id="percentage">%1$d</xliff:g>%% (<xliff:g id="apps">%2$d</xliff:g> apps) de vos applications</string>
+    <plurals name="trackers_presence">
+        <item quantity="one">Présent dans <xliff:g id="percentage">%1$d</xliff:g>%% (<xliff:g id="apps">%2$d</xliff:g> app) de vos applications</item>
+        <item quantity="many">Présent dans <xliff:g id="percentage">%1$d</xliff:g>%% (<xliff:g id="apps">%2$d</xliff:g> apps) de vos applications</item>
```
Comment (Jean-BaptisteC) [inline app/src/main/res/values-fr/strings.xml:92]:
```suggestion
        <item quantity="other">Présent dans <xliff:g id="percentage">%1$d</xliff:g>%% (<xliff:g id="apps">%2$d</xliff:g> apps) de vos applications</item>
```

**Review by Jean-BaptisteC — CHANGES_REQUESTED** [review]
New management keystore failed when, i try to sync project on Android Studio and unable to build debug version because Android Studio didn't find keystore.properties file.

If you don't have a better solution, go back to older releaseConfig

**Comment by PatDyn** [comment]
> New management keystore failed when, i try to sync project on Android Studio and unable to build debug version because Android Studio didn't find keystore.properties file.
> 
> If you don't have a better solution, go back to older releaseConfig


Did you pull the latest changes? Because i refactored the releaseConfig {} so it does not look up the keystore.properties anymore. Also, can you please post logs of the error?

**Review by Jean-BaptisteC — APPROVED** [review]
✅ Pixel 6 - Android 13

**Pull request merged**